src/main/java/ch/njol/skript/classes/data/BukkitClasses.java

```diff
@@ -1565,6 +1566,24 @@ public String toVariableNameString(EnchantmentOffer eo) {
 					"Individual attributes of a snapshot cannot be modified or retrieved.")
 				.requiredPlugins("Minecraft 1.20.2+")
 				.since("INSERT VERSION")
+				.parser(new Parser<EntitySnapshot>() {
+					@Override
+					public boolean canParse(ParseContext context) {
+						return false;
+					}
+
+					@Override
+					public String toString(EntitySnapshot o, int flags) {
+						if (SUPPORTS_GET_AS)
+							return o.getAsString();
+						return o.getEntityType() + " snapshot";
```

Comment:
you could probably lowercase the name and replace _

Reply:
might be able to classes.tostring it in some way too, like via entitydata